content/docs/csidriver/features/powerscale.md

```diff
@@ -565,3 +565,19 @@ When this feature is enabled, the existing `ReadWriteOnce(RWO)` access mode rest
 
 To migrate existing PersistentVolumes to use `ReadWriteOncePod`, please follow the instruction from [here](https://kubernetes.io/blog/2021/09/13/read-write-once-pod-access-mode-alpha/#migrating-existing-persistentvolumes).
 
+## FSGroupPolicy
```

Comment:
@nitesh3108 I don't think we need to mention this under feature since this is already available by default even in the previous release right? Also, we have not implemented anything and we just gave the option to configure with values file
I haven't mentioned this as a feature in csi-powerstore docs.